Xamarin.Essentials/Calendar/Calendar.ios.cs

```diff
@@ -27,16 +27,48 @@ static async Task<IReadOnlyList<ICalendar>> PlatformGetCalendarsAsync()
             return calendarList.AsReadOnly();
         }
 
-        static Task<IReadOnlyList<IEvent>> PlatformGetEventsAsync(string calendarId = null, DateTimeOffset? startDate = null, DateTimeOffset? endDate = null) => throw new NotImplementedException();
-
-        static async Task PlatformRequestCalendarReadAccess()
+        static async Task<IReadOnlyList<IEvent>> PlatformGetEventsAsync(string calendarId = null, DateTimeOffset? startDate = null, DateTimeOffset? endDate = null)
         {
             await Permissions.RequireAsync(PermissionType.CalendarRead);
-        }
 
-        static async Task PlatformRequestCalendarWriteAccess()
-        {
-            await Permissions.RequireAsync(PermissionType.CalendarWrite);
+            var systemAbsoluteReferenceDate = new DateTime(2001, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
I wonder if we should add a link to a apple's doco about this date, as it's not very obvious why it's this exact date

Reply:
done in latest code.